crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -309,8 +309,9 @@ public Response simpleFind(@PathParam(PARAM_ENTITY) String entity,
                                @QueryParam("P") String p,
                                @QueryParam("S") String s,
                                @QueryParam("from") Long from,
-                               @QueryParam("to") Long to) throws IOException {
-        return simpleFind(entity, null, q, p, s, from, to);
+                               @QueryParam("to") Long to,
+                               @QueryParam("maxResults") Long maxResults) throws IOException {
+        return simpleFind(entity, null, q, p, s, from, to, maxResults);
```

Comment:
rest project is using jaxrs 1_1 spec, and that doesn't have beanparam

On Fri, Jun 3, 2016 at 9:08 AM, Burak Serdar <bserdar@ieee.org> wrote:

> I'll look
>
> On Fri, Jun 3, 2016 at 9:06 AM, Brandon Vulaj <notifications@github.com>
> wrote:
>
>> In
>> crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java
>> <https://github.com/lightblue-platform/lightblue-rest/pull/240#discussion_r65722834>
>> :
>>
>> > @@ -309,8 +309,9 @@ public Response simpleFind(@PathParam(PARAM_ENTITY) String entity,
>> >                                 @QueryParam("P") String p,
>> >                                 @QueryParam("S") String s,
>> >                                 @QueryParam("from") Long from,
>> > -                               @QueryParam("to") Long to) throws IOException {
>> > -        return simpleFind(entity, null, q, p, s, from, to);
>> > +                               @QueryParam("to") Long to,
>> > +                               @QueryParam("maxResults") Long maxResults) throws IOException {
>> > +        return simpleFind(entity, null, q, p, s, from, to, maxResults);
>>
>> This list seems to be growing. Does it make sense to replace this list
>> with @BeanParam?
>>
>> https://jersey.java.net/apidocs/latest/jersey/javax/ws/rs/BeanParam.html
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/lightblue-platform/lightblue-rest/pull/240/files/19b956106f5fbd3febd189ab6d88706ae4685595#r65722834>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ADgDDR_APWCK4ctZ2h1w8WtNTIWNwBr2ks5qIELbgaJpZM4IsvZ->
>> .
>>
>
>